samples/subsys/portability/cmsis_rtos_v1/philosophers/prj.conf

```
@@ -11,3 +11,4 @@ CONFIG_POLL=y
 CONFIG_SCHED_SCALABLE=y
 CONFIG_THREAD_CUSTOM_DATA=y
 CONFIG_CMSIS_THREAD_MAX_STACK_SIZE=1024
+CONFIG_SMP=n
```

Let's disable SMP the proper way:

```suggestion
CONFIG_MP_MAX_NUM_CPUS=1
```

---

or filter not to run on SMP enabled platforms?

---

Then we don't get coverage at all though (but at the same time, this doesn't seem very valuable, again it's just a cut/paste of code we have elsewhere, really if we cared we'd just add CMSIS cases to the existing philosophers demo).

---

Actually, given how much chaff we have in CI right now, I'm liking that better.  I'll do that.